src/xo-app/settings/acls/index.js

```diff
@@ -121,10 +132,20 @@ class AclTable extends Component {
   }
 }
 
+@connectStore(() => {
+  return {
+    hosts: createGetObjectsOfType('host'),
+    networks: createGetObjectsOfType('network'),
+    pools: createGetObjectsOfType('pool'),
+    srs: createGetObjectsOfType('SR'),
+    vms: createGetObjectsOfType('VM')
+  }
+})
```

Comment:
Do not use `connectStore()` here as we don't need to subscribe, only access objects on toggle.